[Bug] [API] spelEvaluationException: EL1008E: Property or field 'all' cannot be found

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

```
[ERROR] 2021-12-07 19:02:27.766 org.apache.dolphinscheduler.api.exceptions.ApiExceptionHandler:[46] - 查询worker分组失败
org.springframework.expression.spel.SpelEvaluationException: EL1008E: Property or field 'all' cannot be found on object of type 'org.springframework.cache.interceptor.CacheExpressionRootObject' - maybe not public or not valid?
  at org.springframework.expression.spel.ast.PropertyOrFieldReference.readProperty(PropertyOrFieldReference.java:217)
  at org.springframework.expression.spel.ast.PropertyOrFieldReference.getValueInternal(PropertyOrFieldReference.java:104)
  at org.springframework.expression.spel.ast.PropertyOrFieldReference.getValueInternal(PropertyOrFieldReference.java:91)
  at org.springframework.expression.spel.ast.SpelNodeImpl.getValue(SpelNodeImpl.java:112)
  at org.springframework.expression.spel.standard.SpelExpression.getValue(SpelExpression.java:272)
  at org.springframework.cache.interceptor.CacheOperationExpressionEvaluator.key(CacheOperationExpressionEvaluator.java:104)
  at org.springframework.cache.interceptor.CacheAspectSupport$CacheOperationContext.generateKey(CacheAspectSupport.java:795)
  at org.springframework.cache.interceptor.CacheAspectSupport.generateKey(CacheAspectSupport.java:592)
  at org.springframework.cache.interceptor.CacheAspectSupport.execute(CacheAspectSupport.java:379)
  at org.springframework.cache.interceptor.CacheAspectSupport.execute(CacheAspectSupport.java:345)
  at org.springframework.cache.interceptor.CacheInterceptor.invoke(CacheInterceptor.java:64)
  at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
  at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97)
  at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
  at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:215)
  at com.sun.proxy.$Proxy152.queryAllWorkerGroup(Unknown Source)
  at org.apache.dolphinscheduler.api.service.impl.WorkerGroupServiceImpl.getWorkerGroups(WorkerGroupServiceImpl.java:255)
  at org.apache.dolphinscheduler.api.service.impl.WorkerGroupServiceImpl.queryAllGroup(WorkerGroupServiceImpl.java:233)
```

### What you expected to happen

The query is normal

### How to reproduce

Query the list of workers

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java
+++ b/dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java
@@ -771,4 +771,7 @@ public final class Constants {
      */
     public static final int DRY_RUN_FLAG_NO = 0;
     public static final int DRY_RUN_FLAG_YES = 1;
+
+    public static final String CACHE_KEY_VALUE_ALL = "'all'";
+
 }

--- a/dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/WorkerGroupMapper.java
+++ b/dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/WorkerGroupMapper.java
@@ -17,6 +17,8 @@
 
 package org.apache.dolphinscheduler.dao.mapper;
 
+import static org.apache.dolphinscheduler.common.Constants.CACHE_KEY_VALUE_ALL;
+
 import org.apache.dolphinscheduler.dao.entity.WorkerGroup;
 
 import org.apache.ibatis.annotations.Param;
@@ -40,16 +42,16 @@ public interface WorkerGroupMapper extends BaseMapper<WorkerGroup> {
      *
      * @return worker group list
      */
-    @Cacheable(sync = true, key = "all")
+    @Cacheable(sync = true, key = CACHE_KEY_VALUE_ALL)
     List<WorkerGroup> queryAllWorkerGroup();
 
-    @CacheEvict(key = "all")
+    @CacheEvict(key = CACHE_KEY_VALUE_ALL)
     int deleteById(Integer id);
 
-    @CacheEvict(key = "all")
+    @CacheEvict(key = CACHE_KEY_VALUE_ALL)
     int insert(WorkerGroup entity);
 
-    @CacheEvict(key = "all")
+    @CacheEvict(key = CACHE_KEY_VALUE_ALL)
     int updateById(@Param("et") WorkerGroup entity);
 
     /**

